overrides/config/notenoughwands.cfg

```diff
@@ -231,10 +231,10 @@ wandsettings {
     I:capturing_wand_availability=2
 
     # Add this to the HP * difficultyMult to get the final difficulty scale that affects XP/RF usage (a final result of 1.0 means that the default XP/RF is used)
-    D:capturing_wand_diffcultyAdd=100.0
+    D:capturing_wand_diffcultyAdd=1.0
 
     # Multiply the HP of a mob with this number to get the difficulty scale that affects XP/RF usage (a final result of 1.0 means that the default XP/RF is used)
-    D:capturing_wand_difficultyMult=5.0
+    D:capturing_wand_difficultyMult=0.0
```

Comment:
Why? Wouldn't that make wand use no energy at all?

Reply:
No, that's the default;t values. (cost varies from 10-20k depending on mob type)
It seems that anything higher is calculated in some weird way and while wand has max capacity of 500000 RF, it tries to use 1-20M RF... so you aren't capable of use it at all.
See https://github.com/AllTheMods/ATM-3/issues/491